lib/index.js

```diff
@@ -65,7 +66,8 @@ internals.schema = Joi.object({
         ]),
         index: Joi.boolean(),
         timeout: Joi.number().integer().min(1).allow(false),
-        maxConnectionsPerUser: Joi.number().integer().min(1).allow(false).when('index', { is: true, otherwise: Joi.valid(false) })
+        maxConnectionsPerUser: Joi.number().integer().min(1).allow(false).when('index', { is: true, otherwise: Joi.valid(false) }),
+        verify: Joi.number().integer()
```

Comment:
@hueniverse you mentioned you'd like to see a `min` here? Aside from Joi trickery to achieve that - is there really a need for that?

Reply:
The point of making sure this is not less than the heartbeat rate is to set expectations. If you want to verify the connection faster than the heartbeat check, it will still only happen as fast as messages are passing through or the heartbeat check.

Also, `verify` is not very descriptive. I prefer `minAuthVerifyIntervalMsec`.